Cargo.toml

```diff
@@ -21,9 +21,41 @@ clippy.missing_const_for_fn = "deny"
 clippy.too_long_first_doc_paragraph = "warn"
 
 [workspace.dependencies]
-
-[profile.test.package.proptest]
-opt-level = 3
```

Comment:
I would not like to remove the `protest` optimization levels. Can you add it back towards the end of the file.